packages/app-page-builder-elements/src/createRenderer.tsx

```diff
@@ -1,12 +1,12 @@
 import React from "react";
 import { usePageElements } from "~/hooks/usePageElements";
 import { Renderer, Element } from "~/types";
-import { Theme, StylesObject } from "@webiny/theme/types";
+import { DecoratedTheme, StylesObject } from "@webiny/theme/types";
 import { RendererProvider } from "~/contexts/Renderer";
 import { CSSObject, ClassNames } from "@emotion/react";
 
 interface GetStylesParams {
-    theme: Theme;
+    theme: DecoratedTheme;
```

Comment:
yes, we can discuss that one

Reply:
A relatively easy solution for this would be to rename the `Theme` that we have in the types.ts to, say, `BaseTheme` or `RawTheme`, or whatever. Then, we leave the `DecoratedTheme` as is, but we create a new `Theme = `DecoratedTheme`. That way users would be able to still import `Theme`. 

It's essentially all about DX.

If we were able to go the straightforward way, and simply do renames without of the above manouvers, I can see this PR's number of files also being reduced significantly.